.github/workflows/waspc-ci.yaml

```diff
@@ -163,7 +163,11 @@ jobs:
 
       - name: Ensure examples/todoApp compiles and builds
         if: matrix.os == 'ubuntu-22.04' || matrix.os == 'macos-latest'
-        run: ./examples/todoApp/ensure_app_compiles_and_builds.sh
+        run: ./tools/ensure_app_compiles_and_builds.sh ./examples/todoApp
+
+      - name: Ensure examples/crud-testing compiles and builds
+        if: matrix.os == 'ubuntu-22.04' || matrix.os == 'macos-latest'
+        run: ./tools/ensure_app_compiles_and_builds.sh ./examples/crud-testing
```

Comment:
Nice thinking!

Hm, but could we maybe instead move the Crud stuff to `todoApp`? This goes towards the effort of having one "end all be all" testing app.